certify-pgdb-identity-cp1.properties

```diff
@@ -9,15 +9,22 @@ mosip.certify.vcformat.vc.expiry=true
 # optional field to specify VC signature algo for CertifyIssuer, defaults to Ed25519Signature2020
 # other option: [RsaSignature2018]
 mosip.certify.issuer.vc-sign-algo=Ed25519Signature2020
-mosip.certify.issuer.pub.key=did:web:vharsh.github.io:DID:dev1#key-0
-mosip.certify.issuer.uri=did:web:vharsh.github.io:DID:dev1
+mosip.certify.issuer.pub.key=did:web:vharsh.github.io:DID:devcp1#key-0
+mosip.certify.issuer.uri=did:web:vharsh.github.io:DID:devcp1
 mosip.certify.issuer.svg.template.id=5b9c2a12-810a-7388-2dc8-13ee7ad88bac
-mosip.certify.integration.data-provider-plugin=MockCSVDataProviderPlugin
+mosip.certify.integration.data-provider-plugin=PostgresDataProviderPlugin
 ## CSV specific config
 mosip.certify.data-provider.identifier.column=id
 mosip.certify.data-provider.fields.include=id,name,phoneNumber,dateOfBirth,highestEducation,typeOfHouse,numberOfDependents,works,landArea,landOwnershipType,primaryCropType,secondaryCropType,maritalStatus
 mosip.certify.plugin.csv.file.uri=https://raw.githubusercontent.com/jainhitesh9998/digital-credential-plugins/refs/heads/develop/mock-certify-plugin/src/test/resources/farmer_identity_data.csv
 mosip.certify.issuer.id.field.prefix.url=https://mosip.io/credential/
+
+# Postgres DB specific config
```

Comment:
lets add comments on the property, like what is the property comments, relevance of id filed, what is key and what is value and what is the connect to the template etc..